exporter/clickhouseexporter/documentation.md

```diff
@@ -0,0 +1,79 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# clickhouse
+
+## Internal Telemetry
+
+The following telemetry is emitted by this component.
+
+### otelcol_exporter_clickhouse_sent_logs
```

Comment:
There is overlap with already reported metrics https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/metadata.yaml#L68. Please ensure that we don't have any duplicates